### PR TITLE
FOGL-9004 Python metadata fixes in setup file

### PR DESCRIPTION
**Pull request by ashish-jabble**

### CODE REVIEW

File: python/setup.py [hunk 1]
```diff
@@ -2,11 +2,11 @@
 
 setup(
     name='Fledge',
-    python_requires='~=3.5',
-    version='0.1',
-    description='Fledge',
-    url='http://github.com/fledge/Fledge',
-    author='OSIsoft, LLC',
+    python_requires='>=3.6.9',
+    version='2.5.0',
```
Comment (praveen-garg) [inline python/setup.py:6]: Another case of version bump?

File: python/setup.py [hunk 2]
```diff
@@ -2,11 +2,11 @@
...
+    description='Fledge, the open source platform for the Internet of Things',
+    url='https://github.com/fledge-iot/fledge',
+    author='Dianomic Systems Inc.',
```
Comment (praveen-garg) [inline python/setup.py:9]: I'm not really sure, if We should replace or append?

Dianomic Systems, Inc.

File: python/setup.py (hunk 1 above)
Comment (ashish-jabble) [reply python/setup.py:6]: yupe for consistency across

File: python/setup.py (hunk 2 above)
Comment (ashish-jabble) [reply python/setup.py:9]: @MarkRiddoch Please suggest.

Comment (MarkRiddoch) [reply python/setup.py:9]: We should probably append

Comment (ashish-jabble) [reply python/setup.py:9]: Also please confirm which one we should use for the following:
a) Dianomic Systems
b) Dianomic Systems Inc.
c) Dianomic Systems, Inc.

**Review by praveen-garg — APPROVED** [review]

**Review by MarkRiddoch — APPROVED** [review]

**Pull request merged**